Gopkg.toml

```diff
@@ -95,15 +95,16 @@ required = [
   name = "golang.org/x/oauth2"
   revision = "cdc340f7c179dbbfa4afd43b7614e8fcadde4269"
 
-# TODO why is this overridden?
+# Select the version of knative/pkg we would like to import.
+# This controls when we upgrade apis.
```

Comment:
The removed comment was asking "why is this an override instead of a constraint?"

Can this be a constraint instead?

Reply:
yes! done.